base/compiler/ssair/ir.jl

```diff
@@ -425,6 +428,9 @@ end
     elseif isa(stmt, ReturnNode)
         op == 1 || throw(BoundsError())
         stmt = typeof(stmt)(v)
+    elseif isa(stmt, SSAValue)
+        op == 1 || throw(BoundsError())
+        stmt = typeof(stmt)(v)
```

Comment:
Is this correct? I figured it'd be `stmt=v`

Reply:
Oops, yeah, definitely not correct